app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
```

Comment:
can we make it like this?
`def home; end`
In the case when we don't have to write in the action